src/machine/machine_atsamd21.go

```diff
@@ -644,6 +644,9 @@ func (uart *UART) handleInterrupt(interrupt.Interrupt) {
 	uart.Bus.INTFLAG.SetBits(sam.SERCOM_USART_INTFLAG_RXC)
 }
 
+// check for ensuring we fulfill interface
+var _ i2cController = (*I2C)(nil)
```

Comment:
This seems like a lot of work to add to each file. Can we just have it in the same file as `i2cController`?

Reply:
Seems yes. Done.